src/Symfony/Component/Form/ButtonBuilder.php

```diff
@@ -49,16 +57,36 @@ class ButtonBuilder implements \IteratorAggregate, FormBuilderInterface
      */
     private $options;
 
+    private $dispatcher;
+
     /**
      * @throws InvalidArgumentException if the name is empty
      */
-    public function __construct(?string $name, array $options = [])
+    public function __construct(?string $name, /*EventDispatcherInterface*/ $dispatcher = [], array $options = [])
```

Comment:
I'd suggest `/*EventDispatcherInterface*/ $dispatcher = null` instead. If you pass only the first argument, the default value of the `$this->options` property will be taken from the 3rd argument anyway.